src/main/kotlin/no/nav/syfo/narmesteLederApi/NarmesteLederApi.kt

```diff
@@ -37,6 +38,37 @@ fun Route.registrerNarmesteLederApi(narmesteLederClient: NarmesteLederClient) {
             }
         }
     }
+
+    get("/syfonarmesteleder/sykmeldt/{sykmeldtAktorId}/{orgnummer}") {
```

Comment:
Hvorfor vil du ikke ha orgnummer som en queryParam? Det er vel i prinsippet et filter? 😢

Reply:
Egentlig mest siden endepunktet ble likt som endepunktet for å hente nærmeste leder relasjoner for en nærmeste leder... men la på `sykmeldt` i pathen, så nå er det egentlig. Egentlig mest siden Også beholdt jeg det litt siden @andersostby ikke liker queryparams og siden vi ikke har brukt det andre steder hvor vi har laget tilsvarende queries.

Reply:
Er enig i at det er litt preemptive optimalisering. _MEN_ når vi skal ta i bruk det samme endepunkte i frontenden skal vi hente leder uten orgnummer. Så da må vi kunne spørre uten det også. Men kan godt ta det da..

Reply:
Beklager @MartinMekk, men jeg har latt meg overtale til å bruke queryparams.
Urlen bør derfor være `/syfonarmesteleder/sykmeldt/{sykmeldtAktorId}?orgnummer={orgnummer}`